ussologin/ussologin.go

```diff
@@ -157,5 +162,60 @@ func (f *FileTokenStore) Get() (*usso.SSOData, error) {
 	if err := json.Unmarshal(data, &tok); err != nil {
 		return nil, errgo.Notef(err, "cannot unmarshal token")
 	}
+	if tok.TokenName != name {
+		return nil, nil
```

Comment:
Again, returning (nil, nil) seems wrong to me. Unless specifically documented in the interface, the returned token should be non-nil iff err == nil.